config/initializers/devise.rb

```diff
@@ -14,7 +14,7 @@
   # confirmation, reset password and unlock tokens in the database.
   # Devise will use the `secret_key_base` as its `secret_key`
   # by default. You can change it below and use your own secret key.
-  config.secret_key = Rails.application.secrets.secret_key_base
+  config.secret_key = Rails.application.credentials.secret_key_base
```

Comment:
discussed in person: could we separate this pr into it's own pr and check our credentials have those set correctly

Reply:
Moving this to another PR.. 
Checked all credentials and all have `secret_key_base` set correctly.